### ISSUE TITLE
Data Ingestion features and considerations master page

### ISSUE THREAD

**Issue by tomthetrainer** [deeplearning4j/deeplearning4j#2674]
A few tickets are open that request documentation on Image Transform, Standard Scaler, along with some new image Transform features. This is a good opportunity to organize a large page perhaps with sub pages that would document the data ingest pipeline. We have some really good content on each part, but it is not structured in an easy to find heirarchy, here is our chance to put some structure.

Start with template here,

**Comment by tomthetrainer** [comment]
Rough Draft of outline...

# Data Ingestion pipelines for DeepLEarning 4j

DeepLearning 4J provides the DataVec library for configuring a Data Ingestion pipeline to a Neural Network.

The Data Ingestion Pipeline for  Neural Network can vary greatly from project to project. Here are a few examples.

## Data Sources

CSV data
CSV Sequence Data

Grey Scale Image Data
RGB Image Data
Video Data

## Data Transformation

Normilzation
Standardize

## Data Augmentation

With Image data you can generate additional training data by transforming, sampling or cropping the data to generate additional useful inputs.

## Tools used

Labelling, shuffling and splitting into Train and Test datasets.

(DataVec tools for same)

Normalization Scaler and the like
Convert to what is it, 0 mean StdDEv 1, values between -1 and +1 regardless of wether Farenheit or Celsius might be a good example.



# Examples

See MNist Image Pipeline.

Build example code for the other paths.

## Pages to refer to
https://deeplearning4j.org/datavec Good stuff
https://deeplearning4j.org/opendata
https://deeplearning4j.org/image-data-pipeline.html#record Good stuff
https://deeplearning4j.org/simple-image-load-transform Good Stuff

## Pages to purge modify or clean up along the way..

https://deeplearning4j.org/data-sets-ml
https://deeplearning4j.org/customdatasets
https://deeplearning4j.org/csv-deep-learning

## Notes, take over the NavPanel on Data and ETL

That Navpanel has about 50/50 good rich content and some not so much, Either leave as is and put overview at top, or rework.

**Comment by tomthetrainer** [comment]
Closing this re-open if you have issue with page... https://deeplearning4j.org/Deeplearning4J-ETL-UserGuide
